packages/components/src/components/list-box/_list-box.scss

```diff
@@ -611,9 +611,9 @@ $list-box-menu-width: rem(300px);
 
   // Tweaks for descendants
   // When handling input, we need to target nodes that specifically opt-in to
-  // the `combobox` role in order to make sure the text input is styled
+  // the type text in order to make sure the text input is styled
   // correctly.
-  .#{$prefix}--list-box input[role='combobox'] {
+  .#{$prefix}--list-box input[type='text'] {
```

Comment:
this selector change is breaking light variant styles in the filterable multiselect, so we can either change the role on the input field in the filterable multiselect or change the style rule to apply to both selectors